packages/projects/src/components/ProjectForm.jsx

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) Kernel
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useEffect, useReducer, useState, Fragment } from 'react'
+import { Link } from 'react-router-dom'
+import { useServices, Alert } from '@kernel/common'
+
+import * as runtime from 'react/jsx-runtime.js'
+import { evaluate } from '@mdx-js/mdx'
+import remarkGfm from 'remark-gfm'
+import remarkBreaks from 'remark-breaks'
+import emoji from 'remark-emoji'
+
+import { MDXProvider, useMDXComponents } from '@mdx-js/react'
+
+import { components } from 'constants.js'
```

Comment:
There is some bug/issue with standard.js where it won't let me just import from `'constants'` even though it works. It thinks it is some sort of reserved module name.